docset/winserver2025-ps/smbshare/Reset-SmbServerConfiguration.md

```diff
@@ -143,6 +142,74 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuditClientCertificateAccess
+
+Enables auditing of the clients certificate access attempts on the server. Events are logged when a
```

Comment:
Enables SMB over QUIC client access control audit events. There are three possible events: access allowed, access denied, and error. The access allowed and denied events list properties of the client certificate chain and list any allow and deny access control entries that apply to the client certificates.